llvm/lib/Target/AMDGPU/AMDGPUInstructions.td

```diff
@@ -315,6 +315,20 @@ def srl_16 : PatFrag<
   (ops node:$src0), (srl_oneuse node:$src0, (i32 16))
 >;
 
+def clamp_s16_u8 : PatFrag<
+  (ops node:$src),
+  (i16 (AMDGPUsmed3 $src, (i16 0), (i16 255)))
+>;
+
+def conc_lo_u8_i16 : PatFrags<
```

Comment:
This is just one Pat, Can use PatFrag not PatFrags and avoid the list

Reply:
I'll add another pattern of addition... sorry for the trouble its my first PR....